src/GafferModule/ContextProcessorBinding.cpp

```diff
@@ -53,13 +53,60 @@ using namespace GafferBindings;
 namespace
 {
 
+IECore::InternedString g_inPlugName( "in" );
+IECore::InternedString g_outPlugName( "in" );
```

Comment:
"out" ?

Reply:
How embarrassing. Can I use man flu as an excuse? Fixed in 85eca86.